File: news-angular/src/app/model/NewsModel.ts
import { AuthorModel } from './AuthorModel';
import { NewsLocalizationModel } from './NewsLocalizationModel';
import { TagsModel } from './TagsModel';

export class NewsModel {
  id: String = "";
  tags: Array<TagsModel> = new Array();
  slug: String = "";
  datePublish: Date;
  numberOfVisits: Number;
  author: AuthorModel;
  newsLocalizations: NewsLocalizationModel;
}

File: news_rest/src/main/java/com/example/news_rest/model/NewsModel.java
package com.example.news_rest.model;

import lombok.Data;
import org.springframework.data.annotation.Id;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.util.List;

@Data
public class NewsModel {
    private String id;
    private List<TagsModel> tags;
    private String slug;
    private LocalDate datePublish;
    private Long numberOfVisits;
    private AuthorModel author;
    private NewsLocalizationsModel newsLocalizations;
}
